docker/docker-files-raspbian/prepare-raspbian-img.sh

```diff
@@ -60,11 +60,11 @@ else
 fi
 
 echo "##### Donwloading and extracting..."
-curl -sLfO ${raspbian_url}
+wget -q ${raspbian_url}
 unzip ${raspbian_filename_zip}
 rm ${raspbian_filename_zip}
-curl -sLfO https://raw.githubusercontent.com/dhruvvyas90/qemu-rpi-kernel/master/kernel-qemu-4.14.79-stretch
-curl -sLfO https://raw.githubusercontent.com/dhruvvyas90/qemu-rpi-kernel/master/versatile-pb.dtb
+wget -q https://raw.githubusercontent.com/dhruvvyas90/qemu-rpi-kernel/master/kernel-qemu-4.14.79-stretch
+wget -q https://raw.githubusercontent.com/dhruvvyas90/qemu-rpi-kernel/master/versatile-pb.dtb
```

Comment:
Good finding about TLS key length. I will manually port this commit through another PR so that we can unblock `mender-demo-artifact` PRs